packages/website/src/remark/tabs.js

```diff
@@ -72,38 +68,45 @@ function findTabs(node, index, parent) {
     }
 
     if (child.type === 'comment' && child.value.trim() === '/tabs') {
-      tab.end = index;
+      if (tab) {
+        tab.end = index;
+      }
       break;
     }
   }
 
   return tabs;
 }
 
-function validator(node) {
-  return node.type === 'comment' && node.value.trim() === 'tabs';
-}
-
 function tabs() {
   return root => {
-    slugs.reset();
     let foundTabs = false;
+    let alreadyImported = false;
+
+    visit(root, (node, index, parent) => {
+      if (node.type === 'import') {
+        if (node.value.includes('@theme/Tabs')) {
+          alreadyImported = true;
+        }
+      } else if (node.type === 'comment') {
+        if (node.value.trim() === 'tabs') {
+          const tabs = findTabs(node, index, parent);
 
-    visit(root, validator, (node, index, parent) => {
-      const tabs = findTabs(node, index, parent);
-      const start = tabs[0].start;
-      const end = tabs[tabs.length - 1].end;
+          if (tabs.length > 0) {
+            const start = tabs[0].start;
+            const end = tabs[tabs.length - 1].end;
 
-      if (tabs.length > 0) {
-        foundTabs = true;
-        const newChildren = renderTabs(tabs, parent.children);
-        parent.children.splice(start, end - start, ...newChildren);
+            foundTabs = true;
+            const newChildren = renderTabs(tabs, parent.children);
+            parent.children.splice(start, end - start, ...newChildren);
 
-        return index + newChildren.length;
+            return index + newChildren.length;
+          }
+        }
       }
     });
 
-    if (foundTabs) {
+    if (foundTabs && !alreadyImported) {
```

Comment:
changes done in https://github.com/mccleanp/remark-docusaurus-tabs/pull/2

this package is not yet released